### DISCUSSION TITLE
Is there a way to translate split panes from horizontal to vertical?

### DISCUSSION THREAD

**Discussion by osdiab** [wezterm/wezterm#3876] [Q&A]
Hi, I am coming from Kitty and with that there is a `layout_action rotate` behavior that makes two horizontally split panes become two vertical ones and vice versa. Is there something similar to that in wezterm? `RotatePanes` seems to have a different meaning here so rotate is not the right word I guess. Not a big deal if not but something that I'm used to having so would be nice if I could replicate that, thanks!

**Comment by wez** [comment]
Not currently, no. This issue is a related enhancement request:
* https://github.com/wez/wezterm/issues/3516
